frontend/src/pages/withLayout.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Navbar from '../components/Navbar'
+import Footer from '../components/Footer'
+
+export default Component => props => (
+  <>
+    <Navbar />
+    <Component {...props} />
+    <Footer />
+  </>
+)
```

Comment:
Why does it render itself? Realistically aren't the only props `children` in which case we could just render them directly? Also I think it's odd to call it `Component` since people might mix it up with `React.Component`

Reply:
I'm following the React HOC pattern set out here: https://reactjs.org/docs/higher-order-components.html.

I'll change the naming to WrappedComponent for now.